src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Generated/Models/GatewayExpandOption.cs

```diff
@@ -16,6 +16,8 @@ namespace Microsoft.Azure.Management.ServerManagement.Models
     public enum GatewayExpandOption
     {
         [System.Runtime.Serialization.EnumMember(Value = "status")]
-        Status
```

Comment:
You will need to update the version in project.json and assemblyinfo.cs to release this change.